core/src/main/java/org/keycloak/protocol/oidc/representations/MTLSEndpointAliases.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.keycloak.protocol.oidc.representations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class MTLSEndpointAliases {
```

Comment:
How about backchannel logout endpoint? (not OIDC standard one)

Reply:
@tnorimat Thanks for the suggestion. Since the OIDC WellKnown is supposed to contain only standard data defined in the specifications, my vote is to include just those data. For this Backchannel Logout Endpoint, we don't include it in the OIDC WellKnown endpoint itself and hence IMO it will be good to not include it also in the mtls_endpoint_aliases. Does it makes sense?

In the OIDC Wellknown endpoint, we have only "end_session_endpoint", but that is supposed for the browser logouts and not subject to cert-bound access tokens if I understand correctly?

Reply:
@mposolda Yes, I agree with your opinion.